server/testutils/factories/placementRequestDetail.ts

```diff
@@ -0,0 +1,51 @@
+import { Factory } from 'fishery'
```

Comment:
Could we move this to be the first commit? 
As it's used in the first couple of commits (as things stand) it would be good to see it first.